site/src/pages/workshop/workshop.md

```diff
@@ -375,9 +375,9 @@ export const App: BlockComponent<BlockEntityProperties> = ({
 }) => {
   ...
   // find only the entities in linkedEntities which appear as the destination of one of the projectLinks
-  const 
-  
-  
+  const
```

Comment:
This looks broken anyway? 🤔

Reply:
@Alfred-Mountfield I fixed syntax inside ` ``` ` in https://github.com/blockprotocol/blockprotocol/commit/4d901b9a57f8a462406e781704f641dccf50e672. Feel free to cherry-pick.

Reply:
Cherry-picked and pushed, thanks